README.md

```diff
@@ -1,45 +1,28 @@
-# Webpack Boilerplate
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-A basic webpack boilerplate, ready to be used immediately, no configuration needed. It doesn't tell you, which framework or library you should be using, it contains only a handful of modules:
+# PokeDB Website
 
-- Webpack
-- html-webpack-plugin
-- css-loader
-- style-loader
-- webpack-dev-server
+> This project has a list of pokemons you can like and comment.
 
-Everything else is up to you.
+PokeDB includes the PokeAPI and Involvement API. You can like pokemons and comment on their page, check their type and base stats.
 
```

Comment:
- [ ] I think it would be better for users to get information about your app if you add the following to your Readme file: a 5-minute video explanation about the project, a project screenshot, and a more detailed `Getting started` section with every step of setting up the project.